core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -78,6 +78,8 @@
   private boolean mClosed;
   /** Whether or not the current block should be cached. */
   private boolean mShouldCacheCurrentBlock;
+  /** Include incomplete blocks if Alluxio is configured to store blocks in Alluxio storage. */
```

Comment:
the javadoc is a bit unclear. do you mean:

Whether to store incomplete blocks in Alluxio ?